### PR TITLE
[REVIEW] Allow for running concat against empty dataframes

### PR DESCRIPTION
**Pull request by kkraus14**
Needed for dask_gdf meta calculations.

### CODE REVIEW

**Comment by scopatz** [comment]
LGTM @kkraus14!

File: pygdf/dataframe.py [hunk 1]
```diff
@@ -430,7 +430,7 @@ def _concat(cls, objs, ignore_index=False):
             what = set(o.columns for o in objs)
             raise ValueError('columns mismatch: {}'.format(what))
         # Filter out inputs that have 0 length
```
Comment (kkraus14) [inline pygdf/dataframe.py]: Need to remove this comment.

Comment (dantegd) [reply pygdf/dataframe.py]: you should yell at the guy that created this PR :P

**Comment by kkraus14** [comment]
skip tests

**Comment by dantegd** [comment]
Changes look good  and ready to merge.

**Pull request merged**